src/vm/types/pedersen_instance_def.zig

```diff
@@ -29,27 +29,27 @@ pub const PedersenInstanceDef = struct {
     /// The upper bound on the hash inputs.
     ///
     /// If None, the upper bound is 2^element_bits.
-    _hash_limit: u256,
+    hash_limit: u256,
 
-    pub fn initDefault() !Self {
+    pub fn initDefault() Self {
```

Comment:
what do you think about using zig's ability to set default field values here?

I am thinking:

for values that can be known at comptime, we can set them like so https://ziglang.org/documentation/master/#Default-Field-Values

and then just have `initDefault` return `.{}`

Reply:
Agreed

Reply:
ok, so the style is:

initDefault for structs if and only if theres an allocator involved, otherwise we use zig's default field values

Reply:
so we are waiting for this minor fix @jobez and then merging right? 
@dhruvkelawala I think this is the only little thing left